cpp/src/arrow/util/sse-util.h

```diff
@@ -110,7 +110,11 @@ static inline uint32_t SSE4_crc32_u32(uint32_t crc, uint32_t v) {
 }
 
 static inline uint32_t SSE4_crc32_u64(uint32_t crc, uint64_t v) {
+#if ARROW_BITNESS == 32
+  return 0;
```

Comment:
I would expect a `std::abort` call or equivalent here.

Reply:
Aborting with `std::abort`.

Reply:
Looks like this introduced the following build error...

```
[ 44%] Building CXX object src/arrow/CMakeFiles/arrow_objlib.dir/ipc/reader.cc.obj
[ 44%] Building CXX object src/arrow/CMakeFiles/arrow_objlib.dir/ipc/writer.cc.obj
[ 44%] Built target arrow_objlib
make: *** [Makefile:141: all] Error 2
C:\projects\arrow\cpp\build>goto scriptexit 
```

Unfortunately, this build error does not reproduce in my local environment and further information is not available in appveyor.